### ISSUE TITLE
Currency operations not available.Hitbtc blocked withdraw from my account

### ISSUE THREAD

**Issue by CompetitiveRace** [hitbtc-com/hitbtc-api#138]
Currency operations not available.Hitbtc blocked withdraw from my account
Ticket 246760.What time wait your answer?5 days blocked withdraw!!!Please check it and unblock,i was send photos for AML/KYC verifications 4 day ago!!!

**Comment by CompetitiveRace** [comment]
6days blocked withdraw

**Comment by CompetitiveRace** [comment]
7 days,no resolve!!

**Comment by CompetitiveRace** [comment]
8 days,not resolve,no answer from support!!20 btc blocked!!!ticket #246454

**Comment by Gkathuria2** [comment]
My XEM withdrawal is pending from last 60 hours

Operational id is a58bd9db-2f8f-44f7-b585-dd94913b6712

kindly check these tickets #251471
#251092

Now XEM is offline on receiving address, please cancel withdrawal request and add fund back.

@HitBTC_Support_Team

**Comment by ozakcay** [comment]
Since 1 feb... still waiting.
#224572

**Comment by CompetitiveRace** [comment]
18 days not resolve

**Comment by CompetitiveRace** [comment]
35 days!Not resolve,not answer,account blocked,Balance 20btc

**Comment by dbghekd** [comment]
my account is dbghekd@naver.com
Currency operations not available.Hitbtc blocked withdraw from my account
kindly check these tickets #277698
#274731
What time wait your answer?
5 days blocked withdraw
Please check it and unblock,
i was send photos for AML/KYC verifications

**Comment by spfoo** [comment]
What's going on at hitbtc? I have the same issue since weeks and it's not only for a specific coin. I can buy and sell anything that is in it's trading account. I can deposit to the main account. BUT I can't move any coin - not even BTC - between the main and the trading accounts - either direction.

**Comment by rvd951** [comment]
I m also having the same issue from more than 24 hrs now i can trade but cannot transfer from trading to main or main to trading and even cannot withdraw funds and they are also not bothering to reply .

**Comment by spfoo** [comment]
I finally got a reply fro hitbtc. They said there had been suspicious activity on my account and they suspected it had been hacked so they blocked transfers. They requested I change my password and enabled 2FA - which I had already done. Then get back to them and they'll re-enable the account functionalities.

Of course it's a little strange that I could still buy and sell cryptos on the account if it was blocked...

**Comment by snoopx209** [comment]
Say what? So i should do the same thing ?

⁣Poslano s TypeApp ​

Na 30. tra 2018. 14:46,kod 14:46, spfoo <notifications@github.com> je napisao/la:

**Comment by spfoo** [comment]
Worth a try.

**Comment by snoopx209** [comment]
For how long did u wait fot hitbtc suport to respond to u back?

⁣Poslano s TypeApp ​

Na 30. tra 2018. 15:00,kod 15:00, spfoo <notifications@github.com> je napisao/la:

**Comment by dbghekd** [comment]
about 4~10 days to take me

**Comment by MajZoa** [comment]
were you able to withdraw eventually guys?
I had the same issue today :(

**Comment by MajZoa** [comment]
I transferred BTC from Binance to HitBTC then tried to transfer back to the same address to Binance. Got locked! "Currency operations not available"
Ticket ID #289353 please please look into this!

**Comment by spfoo** [comment]
It's been over a week since I did the change of password and enabled 2FA. I e-mailed support as well that I've done what they asked for. No response and same issue persists: "Currency operations not available". Unbelievable. 
The biggest mistake I've ever made with cryptos was using (s)hitbtc.

**Comment by snoopx209** [comment]
Sam thing here mate.. i lost 0.4 btc

⁣Poslano s TypeApp ​

Na 6. svi 2018. 12:58,kod 12:58, spfoo <notifications@github.com> je napisao/la:

**Comment by Marace** [comment]
Hi,
I have the same issue - I cant withraw/send no coins from my main account on HitBTC. They do not send an confirmation email.  Their support desk asked mi for pictures ( their engineers will solve it o))  )so I sent them pictures but  no response several days......I do not know what to do.....

**Comment by stasenko** [comment]
I have the same problem. "Currency operations not available" Pls, help me. Support ticket ID  #290999

**Comment by Marace** [comment]
I got info that they are working on this issue......no more info I have. Just try to ask support desk again and let me know....
______________________________________________________________
> Od: Aleksandr Stasenko <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 08.05.2018 14:46
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
I have the same problem. "Currency operations not available" Pls, help me. Support ticket ID  #290999

**Comment by stasenko** [comment]
Guys, hitbtc, I've already wrote to You, pls, help. All my coins are "Currency operations not available". Support Ticket ID #291774 and Ticket ID #290999. I'm waiting more than 48 hours for an answer from You. I've already go to verification. Thank You

**Comment by stasenko** [comment]
@Marace , I have no answer from desk support. Do You now anything from this?

**Comment by Marace** [comment]
I am not from hitbtc
______________________________________________________________
> Od: Aleksandr Stasenko <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 09.05.2018 11:11
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
Guys, hitbtc, I've already wrote to You, pls, help. All my coins are "Currency operations not available". Support Ticket ID #291774 and Ticket ID #290999. I'm waiting more than 48 hours for an answer from You. I've already go to verification. Thank You

**Comment by spfoo** [comment]
I haven't heard anything back from them and the problem persists. I think they have serious security problems and they can't solve them. Better not use that service again.

**Comment by spfoo** [comment]
Seems like there are 2 different cases: some where they are doing a KYC-check and others like my case where they only asked me to change my password and enable 2FA.

**Comment by stasenko** [comment]
@MajZoa , how long your are waiting from first support ticket till now? and how long you are waiting for asking KYC documents and origin of deposit?

**Comment by Marace** [comment]
Now you can read on their main page, that they do some maintanance work and thats the reason why some customers have problem with deposit/withrawal. Hope that very soon it should be ok
______________________________________________________________
> Od: spfoo <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 09.05.2018 14:11
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
I haven't heard anything back from them and the problem persists. I think they have serious security problems and they can't solve them. Better not use that service again.

**Comment by stasenko** [comment]
@MajZoa , I try do the same. Please tell where your are writing and what will be finish of your case. Thank You.

**Comment by MajZoa** [comment]
@stasenko
I managed to get their attention on reddit.
I will update in the post I sent earlier once the issue is resolved

**Comment by Marace** [comment]
May ask what is KYC-check? Regards, M.
______________________________________________________________
> Od: spfoo <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 09.05.2018 14:17
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
Seems like there are 2 different cases: some where they are doing a KYC-check and others like my case where they only asked me to change my password and enable 2FA.

**Comment by spfoo** [comment]
The KYC check is a specific procedure for some customers. Probably if you deposit FIAT currencies over a certain amount you could end up in the process. It's basically a legal requirement for any financial institution in order to prevent money laundering. The end result is the same. They block transactions while they check the customer. 

However this is not my case at all. I don't even have much there and it will be 0 once those idiots manage to stumble into each other long enough that by accident my account might be available again.

**Comment by Marace** [comment]
thanks, I just want only move some coins from hitbtc main account to wallet......hope they stop blocking withrawals in the near future...hope that Hitbtc does not go in Bankruptcy...
______________________________________________________________
> Od: spfoo <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 09.05.2018 15:30
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
The KYC check is a specific procedure for some customers. Probably if you deposit FIAT currencies over a certain amount you could end up in the process. It's basically a legal requirement for any financial institution in order to prevent money laundering. The end result is the same. They block transactions while they check the customer.
However this is not my case at all. I don't even have much there and it will be 0 once those idiots manage to stumble into each other long enough that by accident my account might be available again.

**Comment by Marace** [comment]
thanks
______________________________________________________________
> Od: MajZoa <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 09.05.2018 15:20
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
@Marace <https://github.com/Marace>
 KYC = Know Your Customer.
 They ask for identification documents (Passport, Driver's license) and a photo of the customer (selfie in front of the monitor). HitBTC also asks for evidence of source of deposited funds.

**Comment by stasenko** [comment]
@Marace, "Do some maintanance work" this is not my case, I have no ERC20 tokens.
And my ticket Ticket ID #291774 and Ticket ID #290999 still not answering, more than 48 hours.

**Comment by spfoo** [comment]
Actually I'm just trying to pull back some XMR I deposited for a quick sale for BTC there. But I can't even do that. Turns out the "quick sale" I was planning to do takes over a month.

**Comment by stasenko** [comment]
@Marace they start advertising company on coinmarketcap, I don't think they go to Bankruptcy
@MajZoa I'm too have hitBTC's message on reddit that I need KYC after 50 hours, but I send KYC documents already - 2 days ago :( 
My support ticket Ticket ID #291774 and Ticket ID #290999

**Comment by stasenko** [comment]
My all coins locked - "Currency operations not available".
I left two tickets in support - Ticket ID #291774 and Ticket ID #290999.
I wrote two times for verification on compliance@hitbtc.com, send my passport, bank card and photo of me with PC's hitbtc.com.
Only result I have 64 hours on reddit hitbtc wrote that I need go to verification, but I already gone to verification.
I have no choice only go to legal advisors :(

**Comment by stasenko** [comment]
finally, I have answer from support after 4 days of awaiting that I need go to KYC - send photo of ID and selfie. I do this and wait of relock.

**Comment by stasenko** [comment]
u/hitbtc, guys. My support ticket #290999. I'm week - more than 7 days on currency operations not available. 3 days ago I have answer from support that I need send photo of ID and selfie with ticket, and I immediately do this. How long I should wait for relock?

**Comment by Marace** [comment]
I have already several weeks currency operations (withrawals) not available. I think trading within HitBTC is possible, ofcourse its not enough! I hope HitBTC will not steel my coins from my mainaccount!
______________________________________________________________
> Od: Aleksandr Stasenko <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 14.05.2018 13:35
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
u/hitbtc, guys. My support ticket #290999. I'm week - more than 7 days on currency operations not available. 3 days ago I have answer from support that I need send photo of ID and selfie with ticket, and I immediately do this. How long I should wait for relock?

**Comment by stasenko** [comment]
u/hitbtc, thank You for your response. My account is deactivated at 14.07.2018 - verification process was started.

**Comment by Marace** [comment]
hello, HitBTC does not communicate with me, they still block my coins on my main account, they did not tell me any reaoson!! It takes month already....what can I do?? Marek
 
______________________________________________________________
> Od: Aleksandr Stasenko <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 15.05.2018 12:55
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
u/hitbtc, thank You for your response. My account is deactivated at 14.07.2018 - verification process was started.

**Comment by stasenko** [comment]
@Marace , write them on reddit. They answer on reddit. Tag - "hitBTC", "Currency operations not available". Example: https://www.reddit.com/r/hitbtc/comments/83gaj7/currency_operations_not_available_all_cryptos/ or create new topic with tag "hitBTC", "Currency operations not available"

**Comment by Marace** [comment]
thanks, I did it...
______________________________________________________________
> Od: Aleksandr Stasenko <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 18.05.2018 08:35
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
@Marace <https://github.com/Marace> , write them on reddit. They answer on reddit. Tag - "hitBTC", "Currency operations not available". Example: https://www.reddit.com/r/hitbtc/comments/83gaj7/currency_operations_not_available_all_cryptos/ <https://www.reddit.com/r/hitbtc/comments/83gaj7/currency_operations_not_available_all_cryptos/> or create new topic with tag "hitBTC", "Currency operations not available"

**Comment by spfoo** [comment]
My issue was finally resolved. I needed to request a password reset and enable 2FA. It took a little over a month to unblock the account. If you get blocked do those 2 things first - reset your password and enable 2FA if not enabled yet. Depending on your case it's either like mine and that's all you need to do or in the other case you will get a request from hitbtc for some documentation to prove your identity and residency.

**Comment by Marace** [comment]
Hi thanks for sharing your experience. I have 2FA since my registration...I can change my password but I was not told to do it from their support team, no request up to now....they 2x wrote me It would require some time to resolve this issue that their work on it and resolve asap o))
______________________________________________________________
> Od: spfoo <notifications@github.com>
> Komu: hitbtc-com/hitbtc-api <hitbtc-api@noreply.github.com>
> Datum: 23.05.2018 16:21
> Předmět: Re: [hitbtc-com/hitbtc-api] Currency operations not available.Hitbtc
>
My issue was finally resolved. I needed to request a password reset and enable 2FA. It took a little over a month to unblock the account. If you get blocked do those 2 things first - reset your password and enable 2FA if not enabled yet. Depending on your case it's either like mine and that's all you need to do or in the other case you will get a request from hitbtc for some documentation to prove your identity and residency.
